themes/src/main/resources/theme/base/login/template.ftl

```diff
@@ -1,4 +1,4 @@
-<#macro registrationLayout bodyClass="" displayInfo=false displayMessage=true displayRequiredFields=false displayWide=false showAnotherWayIfPresent=true>
+<#macro registrationLayout bodyClass="" displayInfo=false displayMessage=true displayRequiredFields=false showAnotherWayIfPresent=true>
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
@vmuzikar +1, this should work as well

Reply:
@pdrozd Just to be clear. You mean IE should work or the idea of dropping IE should work? :)

Reply:
@vmuzikar @pdrozd Thanks for your comments. 

IE doesn't support `var()` function, so it's not possible to load the variables. In this case, there should be some default hardcoded values in main `.css` file. But, it's contradicted against what @ssilvert suggested above. Then, there will be duplicate properties in  the main `.css` file, in order to keep compatibility with various browsers. However, it's true, the login screen should be displayed properly on all browsers, so I'll add the default values. I just wanted to highlight this issue.

Reply:
Just for the record, here's an [official guide for IE11 support in PF4](https://github.com/patternfly/patternfly/wiki/IE11-Support).

Reply:
@vmuzikar I meant that It would be good if this page also work with IE. The support for IE is dropped, so if this is really complicated, I would not insist on this.

Reply:
I've created some workaround for that and it should be working with IE. But there are another screens, which should be remade then and I think, it's not worth the effort.

Reply:
> @vmuzikar I meant that It would be good if this page also work with IE. The support for IE is dropped, so if this is really complicated, I would not insist on this.

@pdrozd @vmuzikar @mabartos Can we say for sure that IE support is dropped for login screens?  That sounds like a wider discussion.

Reply:
@ssilvert I think we cannot. AFAIK we only dropped IE support in Admin and new Account Consoles. If we want to drop the support for login pages as well, I think it should be blessed by PM and maybe discussed on dev ML as well.

Reply:
I'd say, there could be some basic (not like with PF4) functionality/appearance in login screens and inform customer/user, that the IE support will be dropped out soon and that there is a need for using other browser. I mean, there could be some warning alert with the message. At this time, IMHO, it's not a big deal for customer/user switch to another browser. AFAIK, IE is barely supported by Microsoft, so I think, there is no need for us support that in the big manner either. WDYT?

Reply:
@mabartos +1 but I really think that's not your call and we need to involve PM into the discussion.

Reply:
@vmuzikar I agree. So, what do you suggest, when we'd like to include this in KC 12? Keep some default values, in order to provide, at least, minimal support for IE and then, after discussion, remove/keep the default values? Or is it blocker for this?

Reply:
I see it as a blocker. We still need to count with the possibility that it needs to work in IE. And currently, some pages are broken in IE as you mentioned above, am I correct?

Reply:
I vote for add any hacks needed for the IE support. And create follow-up JIRA "Remove hacks after IE support is removed" for the future :-)

Reply:
@vmuzikar @mposolda Thanks guys! This login screen should work properly with IE, but another screens, which will be edited lately, are probably broken and we'll add the "hack" for solving that. :)

Reply:
@mabartos Do you plan to address the IE issue in this PR or a follow-up?

Reply:
@vmuzikar @mabartos Correct me if I am wrong, but ideally it will be good to include everything in this PR? My point is, that if IE fix is done as a follow-up, then there will be some time period during which the Keycloak master will be broken with IE. Hence there is a risk of scenario like:
1) This PR is merged . KEycloak master will become broken with IE
2) Follow-up PR is send for fix IE
3) Keycloak 12 is released
4) PR from step 2 is merged. But it is late as Keycloak 12 was already released with broken IE...

(same can happen with steps 2 or 3 switched)

Reply:
@mposolda +1

Reply:
> The page is completely broken in IE. We're starting to drop IE support, but not sure if we're ready to do so on something as elementary as the login screen. 

@mabartos (Cc: @vmuzikar @mposolda) after adding the workaround the screen looks good even on IE11:
<img width="1785" alt="image" src="https://user-images.githubusercontent.com/12138171/96992772-9eaed600-152a-11eb-9a80-62703e78dbbf.png">

Reply:
If this PR works correctly with IE, there won't be error with another screens. PF3 and PF4 work simultaneously, so some of the additional PRs don't have to be merged, if they have some issue with IE yet. The additional PRs only upgrade the components, which look, in most cases, very similar. This PR will work with IE, and I think, it's sufficient so far. So there won't be any follow-up task for IE workaround. Everything will be included in each PR.

Reply:
@Pepo48 @mabartos From my PoV it feels kinda broken in IE. Just compare the following screenshots (left is PF3 in IE, right PF4 in IE):
![image](https://user-images.githubusercontent.com/13906855/97010006-21dd2580-1545-11eb-99e7-3f348548e0c4.png)
![image-2](https://user-images.githubusercontent.com/13906855/97010010-23a6e900-1545-11eb-9ed1-b4e8f5718bd1.png)
Notice the input fields which I think are missing some padding, notification looks akward. And it's possible there'll be some more broken components. I think it's definitely a downgrade in UX for IE users.

But I don't want to block this so I'll leave the decision to others if this is good enough or not.

Reply:
I'll try to remake it, but I can't spend a lot of time on that.

Reply:
@mabartos Thanks!

Reply:
It isn't like on Chrome or other browsers, but I think, it's sufficient. WDYT @vmuzikar ?
![ieSignIn](https://user-images.githubusercontent.com/38039883/97168741-40336300-1789-11eb-8b97-3b79a5e3d44f.png)

Reply:
@mabartos Thank you for another batch of fixes! Nice job! I know IE can be very frustrating sometimes...

The only issue I see is the notifications and how the icon is somewhat misaligned:
![Capture](https://user-images.githubusercontent.com/13906855/97201890-3cb4d180-17b3-11eb-97f0-2466f976e0ea.PNG)

But again, I don't want to block this PR because of this. I'll leave it to other if we can live with this minor UI glitch.

Reply:
@vmuzikar Thanks for your review. I know, there is some issue with register alert (some alert with longer message). But in the follow-up task with registration, there won't be this alert, but the errors will be under the inputs. Otherwise, the simple alert looks almost normal. What about edit these minor changes in follow-up task, which I'm currently working on in order to not block other PRs, which depend on this PR?

Reply:
I think the misaligned notification icons affect all notifications, not just alerts with a longer message. It's apparent on your screenshot above too.

Reply:
Yes, you're right. All notifications.

Reply:
I was able to edit the notifications. At this moment, I think, the IE compatibility is solved for this PR. So, IMO, it could be finally merged. WDYT @vmuzikar ? :) 

![ieSignInSuccess](https://user-images.githubusercontent.com/38039883/97303066-3299de00-185a-11eb-9c3a-83807eff3ee2.png)
![ieREgisterErr](https://user-images.githubusercontent.com/38039883/97303068-33cb0b00-185a-11eb-960e-c6006c25ac4f.png)